src/main/kotlin/calculator/DelimiterParser.kt

```diff
@@ -0,0 +1,24 @@
+package calculator
+
+object DelimiterParser {
+
+    private val DEFAULT_DELIMITER = listOf(",", ":")
+    private val CUSTOM_DELIMITER_PATTERN = Regex("""//(.*)\\n(.*)""")
+
+    fun extractDelimiters(input: String): Delimiters {
+        val delimiters = Delimiters()
+        delimiters.add(DEFAULT_DELIMITER)
+
+        CUSTOM_DELIMITER_PATTERN.find(input)?.let {
+            delimiters.add(it.groupValues[1])
+        }
+
+        return delimiters
```

Comment:
일단 비어있는 Delimiters를 선언한 뒤, 하나하나 추가하는 것 보다. 
미리 Delimiters 객체를 생성하기위해 필요한 값들을 만든 뒤에, 생성자를 통해 단순히 주입해주도록 변경해보면 어떨까요? 😃

객체가 생성된 이후, 별다른 준비과정없이 자신의 책임을 수행하기만하면 되는 상태로 만들어주세요. 🙏 (PositiveNumbers 또한 비슷한 상황인 것 같습니다.)

```suggestion
        fun extractDelimiters(input: String): Delimiters {
        val customDelimiterMatchResult = CUSTOM_DELIMITER_PATTERN.find(input)
        val delimiters = if (customDelimiterMatchResult != null) {
            DEFAULT_DELIMITER + customDelimiterMatchResult.groupValues[1]
        } else {
            DEFAULT_DELIMITER
        }

        return Delimiters(delimiters)
    }
```